### PR TITLE
meta/processor/index change log level

### PR DESCRIPTION
**Pull request by Nivras**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### What problem(s) does this PR solve?
Issue(s) number: 

Description:
change LOG(ERROR) to LOG(INFO), and VLOG(3) to VLOG(1) in meta/processor/index

#### Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: src/meta/processors/index/GetEdgeIndexProcessor.cpp [hunk 1]
```diff
@@ -16,8 +16,8 @@ void GetEdgeIndexProcessor::process(const cpp2::GetEdgeIndexReq& req) {
   auto edgeIndexIDRet = getIndexID(spaceID, indexName);
   if (!nebula::ok(edgeIndexIDRet)) {
     auto retCode = nebula::error(edgeIndexIDRet);
-    LOG(ERROR) << "Get Edge Index SpaceID: " << spaceID << " Index Name: " << indexName
-               << " failed, error: " << apache::thrift::util::enumNameSafe(retCode);
+    LOG(INFO) << "Get Edge Index SpaceID: " << spaceID << " Index Name: " << indexName
```
Comment (liwenhui-soul) [inline src/meta/processors/index/GetEdgeIndexProcessor.cpp:19]: get/list operation is not schema change, it is supposed to use VLOG

File: src/meta/processors/index/GetTagIndexProcessor.cpp [hunk 2]
```diff
@@ -42,7 +42,7 @@ void GetTagIndexProcessor::process(const cpp2::GetTagIndexReq& req) {
 
   auto item = MetaKeyUtils::parseIndex(nebula::value(indexItemRet));
   if (item.get_schema_id().getType() != nebula::cpp2::SchemaID::Type::tag_id) {
-    LOG(INFO) << "Get Tag Index Failed: Index Name " << indexName << " is not TagIndex";
+    VLOG(2) << "Get Tag Index Failed: Index Name " << indexName << " is not TagIndex";
```
Comment (liwenhui-soul) [inline src/meta/processors/index/GetTagIndexProcessor.cpp:45]: I don't understand, is this an error log?

Comment (pengweisong) [reply src/meta/processors/index/GetTagIndexProcessor.cpp:45]: look likes it an error indeed.

**Comment by Sophie-Xie** [comment]
pengwei fixed it.

**Pull request closed without merging**